x/ccv/provider/keeper/misbehaviour.go

```diff
@@ -101,20 +101,28 @@ func (k Keeper) GetByzantineValidators(ctx sdk.Context, misbehaviour ibctmtypes.
 	// and return the intersection of validators who signed both
 
 	// create a map with the validators' address that signed header1
-	header1Signers := map[string]struct{}{}
-	for _, sign := range lightBlock1.Commit.Signatures {
-		if !sign.ForBlock() {
+	header1Signers := map[string]int{}
+	for idx, sign := range lightBlock1.Commit.Signatures {
+		if sign.Absent() {
```

Comment:
I think we should stick to `sign.ForBlock()` and not change back to `sign.Absent()` even if we verify the signatures. There is no reason to verify `nil` signatures because i) potentially reduce time, ii) we're inconsistent with what `VerifyCommitLightTrusting` is checking.